Mathlib/NumberTheory/EllipticDivisibilitySequence.lean

```diff
@@ -109,49 +114,129 @@ private def normEDS'' (b c d : R) : ℕ → R
     have h2 : m + 2 < n + 5 := (lt_add_one _).trans h3
     have h1 : m + 1 < n + 5 := (lt_add_one _).trans h2
     if hn : Even n then
-      normEDS'' b c d (m + 4) * normEDS'' b c d (m + 2) ^ 3 * (if Even m then b ^ 4 else 1) -
-        normEDS'' b c d (m + 1) * normEDS'' b c d (m + 3) ^ 3 * (if Even m then 1 else b ^ 4)
+      preNormEDS' b c d (m + 4) * preNormEDS' b c d (m + 2) ^ 3 * (if Even m then b else 1) -
+      preNormEDS' b c d (m + 1) * preNormEDS' b c d (m + 3) ^ 3 * (if Even m then 1 else b)
```

Comment:
```suggestion
        preNormEDS' b c d (m + 1) * preNormEDS' b c d (m + 3) ^ 3 * (if Even m then 1 else b)
```
From the [Style Guide](https://leanprover-community.github.io/contribute/style.html): "This rule, i.e., indent an additional two spaces, applies more generally whenever a term spans multiple lines."